doc/edl/README.md

```diff
@@ -159,6 +159,26 @@ implementing custom resources:
 We will support TPR first, because some of our clusters is using
 Kubernetes v1.6. CRD will be supported in the future.
 
+In the near feature, we will use CRD to replace TPR to define our training job resource,
+then the resource defined under `go/edl/resource` will be deprecated.
+If you want to use TPR in Kubernetes with low version,
+please checkout project to the tag `unreleased-tpr`.
+
+Currently, implementation with CRD is still under development,
+you cloud use the following command to check and generate dependent codes:
+```
+# check freshness of generated codes
+$ scripts/verify-codegen.sh
+
+# update dependent codes
+$ scripts/update-codegen.sh
+```
+For more detailed operation, please refer to [article](https://blog.openshift.com/kubernetes-deep-dive-code-generation-customresources/).
+When the definition of resource is stable, we will commit generated dependenct codes.
+Just skip `go/pkg` when running `go test` during your development.
```

Comment:
I think we are removing `go/pkg` directory now?

Reply:
yes, but we still have this currently. When I fix conflicts in #587 I will change the package name here.